### ISSUE TITLE
Bug in detecting Markers

### ISSUE THREAD

**Issue by vgenten** [AltranResearchMedica/ARAM#6]
Hello,

I had an issue running the markerdetection.
When I tried to run the detection of my generated ARAM tag, nothing happend and no tag was detected. When I covered a part of the tag sheet the tags were detected correctly. But then I covered part of the sheet in a static mode, like a peace of paper covering one marker, and again nothing worked.
Do you have any idea why this is happening? A closer look at the code from ARAM library unfortunately didn't bring me any further.

I would really appreciate if you can help me with this issue.

**Comment by akornmann** [comment]
What do you mean by markerdetection ? Single marker or a grid of marker ?

Your generated ARAM tag are correctly linked to your application ? Is your "generatedMarker.xml" well placed ?

**Comment by vgenten** [comment]
Generally I got both examples working, and generated my own markers, of which I passed the .xml file to the library (my suggestion here would be to set it via an api)
Maybe the following images make my problem clearer:

![detection](https://user-images.githubusercontent.com/24294021/42943749-ee35c9ac-8b63-11e8-9e1c-3a3e08de19f3.png)

![detection2](https://user-images.githubusercontent.com/24294021/42943766-fa31a65e-8b63-11e8-9e4c-8199cd83f9f0.png)

Same thing happens When I try to detect a single marker and partially occlude the white margin.

**Comment by akornmann** [comment]
You need some white margin to detect black border of tag.

But on the picture, everything looks ok for me, the two hiddens tags cannot be detected by ARAM.

**Comment by vgenten** [comment]
Thats what I want to figure out, how big the margin has to be.
Yes the second image shows good results, but why isn't there any detection in the first image?
It's the same marker grid...

**Comment by akornmann** [comment]
ok i think it was a sample of your grid !
I think at least big as your black border is ok.

I really have no clue for the first one. Is there any exception thrown ? (Are you in debug mode)

**Comment by vgenten** [comment]
Yes I debugged, but I just get no markers detected and the loop continues...
So if you're running the example like "First App" you always get a detection of the markers? Even if they are completely still and the marker or the camera is not moving?

**Comment by akornmann** [comment]
It's a long time I didnt work on it, sorry i'm not sure :(
